site/interfaces/index.ts

```diff
@@ -28,6 +28,12 @@ export interface CookiesApiRequest extends NextApiRequest {
     };
 }
 
+export interface DeleteUserApiRequest extends NextApiRequest {
```

Comment:
I would just declare this in the api file if it's only used there

Reply:
done